app/src/main/cpp/skyline/soc/gm20b/engines/maxwell_dma.h

```diff
@@ -200,15 +201,15 @@ namespace skyline::soc::gm20b::engine {
                 u8 numDstComponentsMinusOne : 2;
                 u8 _pad6_ : 6;
 
-                u8 ComponentSize() {
+                inline const u8 ComponentSize() const {
                     return componentSizeMinusOne + 1;
                 }
 
-                u8 NumSrcComponents() {
+                inline const u8 NumSrcComponents() const {
                     return numSrcComponentsMinusOne + 1;
                 }
 
-                u8 NumDstComponents() {
+                inline const u8 NumDstComponents() const {
```

Comment:
No need to return a `const` type, we're *fine* with the functions being `const` depending on the situation but `inline` should be removed as it's a useless semantics here at least as far as I can remember, @bylaws can confirm.